packages/block-library/src/calendar/index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Server-side rendering of the `core/calendar` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/calendar` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the block content.
+ */
+function render_block_core_calendar( $attributes ) {
+	global $monthnum, $year;
+	if ( isset( $attributes['monthNum'] ) ) {
+		$monthnum = $attributes['monthNum'];
+	}
+
+	if ( isset( $attributes['year'] ) ) {
+		$year = $attributes['year'];
+	}
+	$custom_class_name = empty( $attributes['className'] ) ? '' : ' ' . $attributes['className'];
+	$align_class_name  = empty( $attributes['align'] ) ? '' : " align{$attributes['align']}";
```

Comment:
nit: Odd that we're inconsistent here with how we prepend the space.

```suggestion
	$align_class_name  = empty( $attributes['align'] ) ? '' : ' ' . align{$attributes['align']};
```